### DISCUSSION TITLE
Feature: Add option to hide pinned notes

### DISCUSSION THREAD

**Discussion by RonanCodes** [standardnotes/forum#2687] [Suggestions] [answered]
Add the option to hide pinned notes like on the desktop app.

I often just want to look at my recent note and have to scroll all the way down past the pinned notes.

**Comment by JaspalSuri** [comment, marked as answer]
Hi @RonanCodes, thanks for the suggestion! If you're subscribed to Extended, you can create a [Dynamic Filter / Smart Tag](https://standardnotes.com/help/42/how-do-i-view-a-list-of-untagged-notes-and-create-other-dynamic-filters) in the web or desktop apps (that will then be accessible across all apps) that allows you to hide pinned notes.

To create a Dynamic Filter that hides pinned notes, please proceed with the following: 
1. Open the **Extensions** menu on the desktop/web app, then install & activate the **Folders** extension.
2. Within the **Folders** extension, click on **Views**, then press the **+** button.
3. Copy and paste the follow smart tag syntax:
`!["Not Pinned", "pinned", "=", false]`
4. Press enter on your keyboard.

At this point, you should see an item called "**Not Pinned**" under **Views**. You can select this item to view filter out your pinned notes. 🙂
